ui/app/common/bifrost/bifrostConfig.js

```diff
@@ -26,11 +26,16 @@
   module.provider('bifrostConfig', function () {
 
     let bifrostUrl;
+    let bifrostRestUrl;
```

Comment:
still invalid naming?

I thought there should be two variables likes this:

```
bifrostUrl ---> externalBifrostUrl
bifrostWsUrl ---> bifrostWsUrl
```

Reply:
Where is method, for example `setBifrostRestUrl` executed? Why it's implemented if it's not used.

Reply:
removed and fixed